### PR TITLE
Added tailwind svelte component framework

### PR DESCRIPTION
**Pull request by Octopus773**

### CODE REVIEW

File: web/src/lib/components/tabs.svelte [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<script context="module" lang="ts">
```
Comment (zoriya) [inline web/src/lib/components/tabs.svelte:1]: delete this unused file

File: web/src/lib/views/FeedsList.view.svelte [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<script lang="ts">
```
Comment (zoriya) [inline web/src/lib/views/FeedsList.view.svelte:1]: useless file

File: web/src/routes/+layout.svelte [hunk 3]
```diff
@@ -0,0 +1,23 @@
+<script>
```
Comment (zoriya) [inline web/src/routes/+layout.svelte:1]: again

File: web/src/lib/components/tabs.svelte (hunk 1 above)
Comment (Octopus773) [reply web/src/lib/components/tabs.svelte:1]: In due time !

**Review by zoriya — APPROVED** [review]

**Pull request merged**